Client/Frontend/UserContent/UserScripts/PlaylistSwizzler.js

```diff
@@ -0,0 +1,12 @@
+
+// Stub out the MediaSource API so video players do not attempt to use `blob` for streaming
+if (window.MediaSource || window.WebKitMediaSource || window.HTMLMediaElement && HTMLMediaElement.prototype.webkitSourceAddId) {
+    
+    window.MediaSource = null;
+    window.WebKitMediaSource = null;
+    //HTMLMediaElement.prototype.webkitSourceAddId = null;
+    //window.SourceBuffer = null;
```

Comment:
these comments still needed?

Reply:
This one I want to keep because it may in the future be added to iPhone when the `MediaSource` is official API and no longer a draft. I left it here so that I remember what it is in webkit that handles that stuff. That way, in the future, I can just google the webkitSourceAddId if I don't remember OR I can uncomment it and see the effects IF it exists then.